docs/architecture/cloud-native/index.md

```diff
@@ -39,15 +39,17 @@ All other marks and logos are property of their respective owners.
 
 Authors:
 
-> **Steve "ardalis" Smith** - Software Architect and Trainer - [Ardalis.com](https://ardalis.com)
->
 > **Rob Vettor** - Microsoft - Principal Cloud System Architect/IP Architect - [thinkingincloudnative.com](http://thinkingincloudnative.com/about/)
+>
+> **Steve "ardalis" Smith** - Software Architect and Trainer - [Ardalis.com](https://ardalis.com)
 
 Participants and Reviewers:
 
 > **Cesar De la Torre**, Principal Program Manager, .NET team, Microsoft
```

Comment:
Not sure if we should update Cesar's team here as well

Reply:
Done